packages/shortest/src/ai/tools/index.ts

```diff
@@ -29,6 +29,42 @@ export const AITools = [
       required: ["action", "username", "password"]
     }
   },
+  {
+    name: "render_email",
+    description: "Fetch, decode and render received email in browser",
+    input_schema: {
+      type: "object",
+      properties: {
+        action: {
+          type: "string",
+          enum: ["render_email"],
+          description: "Render the received email in a new tab"
+        }
+      },
+      required: ["action"]
+    }
+  },
+  {
+    name: "sleep_milliseconds",
+    description: "Pause test execution for specified duration",
+    input_schema: {
+      type: "object",
+      properties: {
+        action: {
+          type: "string",
+          enum: ["sleep_milliseconds"],
+          description: "The action to perform"
+        },
+        duration: {
+          type: "number",
+          description: "Duration to sleep in milliseconds (e.g. 5000 for 5 seconds)",
+          minimum: 0,
+          maximum: 60000
+        }
+      },
+      required: ["action", "duration"]
+    }
+  },
```

Comment:
Added two new tools here: render_email and sleep_miliseconds 

render_email is used to execute Mailosaur email validation in `browser-tools.ts` file. 
`sleep_miliseconds` is used to help AI to wait for a condition to be met. Either instructed by the user or based on AI judgement.

Why both of them needed in this PR: 
getting the latest email requires AI to wait for a few seconds to so that the user receives the email, otherwise the test case might fail prematurely